proto/tendermint/types/types.pb.go

```diff
@@ -29,7 +29,7 @@ var _ = time.Kitchen
 // proto package needs to be updated.
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
-// BlockIdFlag indicates which BlockID the signature is for
+// BlockIdFlag indicates which BlcokID the signature is for
```

Comment:
I have a feeling this is the one line change that's causing the generated code checker to fail which I think is fine. I think we just need to update the documentation on the `.proto` file.